pymc/tests/test_distributions.py

```diff
@@ -1729,9 +1730,15 @@ def test_poisson(self):
             {"mu": Rplus},
         )
 
+    def test_diracdeltadist(self):
+        check_logp(DiracDelta, I, {"c": I}, lambda value, c: np.log(c == value))
+        check_logcdf(DiracDelta, I, {"c": I}, lambda value, c: np.log(value >= c))
+
     def test_constantdist(self):
-        check_logp(Constant, I, {"c": I}, lambda value, c: np.log(c == value))
-        check_logcdf(Constant, I, {"c": I}, lambda value, c: np.log(value >= c))
+        with pytest.warns(FutureWarning):
```

Comment:
```suggestion
        with pytest.warns(FutureWarning, match="..."):
```

Reply:
This test can be at the end of the file, it shouldn't be in this generic class that checks for logp/logcdf stuff